app/views/bulk_submissions/index.html.erb

```diff
@@ -7,23 +7,33 @@
     <%= govuk_table do |table|
       table.with_head do |head|
         head.with_row do |row|
-          row.with_cell(text: "Filename", header: true) if Rails.env.development?
           row.with_cell(text: t(".created_at"), header: true)
           row.with_cell(text: t(".expires_at"), header: true)
+          row.with_cell(text: t(".filename"), header: true)
           row.with_cell(text: t(".status"), header: true)
           row.with_cell(text: t(".action"), header: true)
+          row.with_cell(text: "")
```

Comment:
Accessibility/Improvement: 

Columns can be made to stretch across multiple rows with `colspan` parameter, therefore, you could remove the empty head cell and apply the `colspan` parameter to the previous sibling.

for example: `row.with_cell(text: t(".action"), header: true, colspan: 2, scope: 'colgroup')`